storybookjs/storybook
# Vue: fixed decorators being added multiple times

Issue: #4233

## What I did
Added a check for default decorator `withSubscriptionTracking` and if it is, return that decorated story, since `withSubscriptionTracking` return story with all previous decorators and there is no need to call that once again inside `createWrapperComponent`.
## How to test

- Is this testable with Jest or Chromatic screenshots?
Yes. There are already two tests suites:
-`examples/vue-kitchen-sink/src/stories/addon-centered.stories.js`
-`examples/vue-kitchen-sink/src/stories/custom-decorators.stories.js`
which are testing decorators behavior. You can see the difference by looking at snapshots diff, in the previous one there were the same decorators wrapping story twice.

- Does this need a new example in the kitchen sink apps?
No.
- Does this need an update to the documentation?
No.

### Not related to the issue
I brought back `examples/cra-kitchen-sink/src/stories/index.stories.js` `App` import to the previous version because it was failing core tests and linting straight from clean repo.
